NVorbis/NewStreamEventArgs.cs

```diff
@@ -16,20 +10,20 @@ namespace NVorbis
     public class NewStreamEventArgs : EventArgs
     {
         /// <summary>
-        /// Creates a new instance of <see cref="NewStreamEventArgs"/> with the specified <see cref="IPacketProvider"/>.
+        /// Creates a new instance of <see cref="NewStreamEventArgs"/> with the specified <see cref="IStreamDecoder"/>.
         /// </summary>
-        /// <param name="packetProvider">An <see cref="IPacketProvider"/> instance.</param>
-        public NewStreamEventArgs(IPacketProvider packetProvider)
+        /// <param name="streamDecoder">An <see cref="IStreamDecoder"/> instance.</param>
+        public NewStreamEventArgs(IStreamDecoder streamDecoder)
         {
-            if (packetProvider == null) throw new ArgumentNullException("packetProvider");
+            if (streamDecoder == null) throw new ArgumentNullException("packetProvider");
```

Comment:
The argument name in the exception needs to be updated. Some properties here can be made read only as well if you want but it's not a big deal.

Reply:
I have this corrected in my local copy, so it'll be fixed when I push the last batch of changes here.